main.py

```diff
@@ -5,14 +5,36 @@
 Group: J
 '''
 
-def hello():
-  print("Hello World! TEST NEW FEATURE")
-  print("testing a feature specific 2")
-  return
+class BasicML:
+    '''Initialized with a 100 word memory and a single
+    register named accumulator, as well as a dictionary
+    for the opcodes'''
+    def __init__(self):
+        self.memory = ["+0000"]*100
+        self.accumulator = "+0000"
+        self.pointer = 0
```

Comment:
What is this for?

Reply:
The pointer variable is for iterating the memory location. It starts at memory location 0. Currently, there isn't any function or method that uses it.

Reply:
The branch methods will directly modify the pointer.